Fix typo in SWOT Analysis

lunohodov opened the pull request:
This PR fixes a small typo in SWOT Analysis.

Closes #119.

lunohodov committed:
Fix typo in SWOT Analysis

lunohodov commented on sprint-guide/exercises/swot-analysis.md:
```diff
@@ -44,7 +44,7 @@ Weaknesses are things that we consider the company or product to be lacking in o
 - What do the target audience complain about or criticise?
 
-For example: "Long waiting times", "lack of persoanlity", "Users complain about lost items" are all weaknesses.
+For example: "Long waiting times", "lack of personality", "Users complain about lost items" are all weaknesses.
```
I am not sure I understand the capitalization rules used for the examples – they appear inconsistent to me. Shouldn’t it be “Lack of personality” instead?

cpytel replied:
Yeah, standardizing the capitalization by capitalizing that L would be an improvement. Thanks.

lunohodov replied:
Great!

The examples in other sections follow a similar pattern. I capitalized them. In addition, the lead-in sentence for each question list felt repetitive and “these” was used twice referring to different subjects.

Let me know, if a change does not align and I'll drop it.

cpytel replied:
This looks good, thanks for your contribution!

lunohodov committed:
Fix more typos in SWOT Analysis

lunohodov committed:
Standardize capitalization within SWOT examples

lunohodov committed:
Reduce repetitiveness

The lead-in sentence for each question list feels repetitive
and “these” is used twice referring to different subjects.

This commit rewords the lead-in sentences.
